samples/react-chat/src/App.tsx

```diff
@@ -28,8 +28,9 @@ const ChatListWrapper = memo(({ onSelected }: { onSelected: (e: GraphChat) => vo
   const onAllMessagesRead = useCallback((chatIds: string[]) => {
     console.log(`Number of chats marked as read: ${chatIds.length}`);
   }, []);
-  const onLoaded = () => {
-    console.log('Chat threads loaded.');
+  const onLoaded = (num_threads: number) => {
```

Comment:
for consistency, I recommend not using the _ for naming variables since we don't use it anywhere else in general.